source/aggregation-tutorials/one-to-one-join.txt

```diff
@@ -0,0 +1,208 @@
+.. _node-aggregation-one-to-one:
+
+===============
+One-to-One Join
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, lookup, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to perform a one-to-one join using
```

Comment:
Nit: Voice should be directive related to [this style guide link on verbs](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately). 
```suggestion
In this tutorial, you will learn how to perform a one-to-one join using
```